### DISCUSSION TITLE
no onboarding so what are default login credentials?

### DISCUSSION THREAD

**Discussion by dkebler** [nextcloud/all-in-one#518] [Questions] [answered]
Guess I missed something here.  Got into the master container, got all the containers downloaded and running.  Click on the `open your  nextcloud` and was expecting some kind of onboarding.  Instead it's asking for login credentials?  Looking around the repo I see no mention of a default admin login.    

What would be the default admin username/pw?

**Comment by dkebler** [comment, marked as answer]
super duh,  it's in a dropdown above that link/button.

**Comment by szaimen** [comment]
Hi, thanks for your report! We'll make it more visible with https://github.com/nextcloud/all-in-one/pull/520

**Reply by szaimen** [reply]
Available since https://github.com/nextcloud/all-in-one/releases/tag/v1.1.0
